onnx/helper.py

```diff
@@ -1563,6 +1563,16 @@ def tensor_dtype_to_storage_tensor_dtype(tensor_dtype: int) -> int:
     """
     return mapping.TENSOR_TYPE_MAP[tensor_dtype].storage_dtype
 
+def tensor_dtype_to_storage_np_dtype(tensor_dtype: int) -> int:
```

Comment:
Do we need this to be public?

Reply:
Good question. I guess it could be argued that if a user were creating a raw tensor then having that information could help in determining whether or not the storage has the correct number of elements.